### PR TITLE
Sprint16 code reviews refactor changes

### PR DESCRIPTION
**Pull request by Jeyaprakash-NK**
1. type 'any' fixes
2. prettier fixes
3. setClustersList name change

### CODE REVIEW

File: src/gcs/gcsBrowserWidget.tsx [hunk 1]
```diff
@@ -102,10 +102,14 @@ export class GcsBrowserWidget extends Widget {
...
-          if(content.items && content.items.length > 0) {
+          if (content.items && content.items.length > 0) {
             const result = await showDialog({
               title: 'Upload files',
-              body: file.name + ' already exists in ' + path.bucket +' and click ok to overwriting file',
+              body:
+                file.name +
+                ' already exists in ' +
+                path.bucket +
+                ' and click ok to overwriting file',
```
Comment (Shubha-accenture) [inline src/gcs/gcsBrowserWidget.tsx:112]: Please modify the message 'and click ok to overwriting file' to: 'Do you want to overwrite the file?'

**Review by Shubha-accenture — CHANGES_REQUESTED** [review]

File: src/gcs/gcsBrowserWidget.tsx (hunk 1 above)
Comment (Jeyaprakash-NK) [reply src/gcs/gcsBrowserWidget.tsx]: Resolved

File: style/base.css [hunk 2]
```diff
@@ -202,8 +202,7 @@ body[data-jp-theme-name='JupyterLab Dark'] .job-cancel-button-style:hover {
 }
 .css-uqjlkp-MuiFormControl-root-MuiTextField-root .MuiInputBase-root {
   padding: unset !important;
-  }
-  .css-uqjlkp-MuiFormControl-root-MuiTextField-root .MuiInputBase-root input {
+}
+.css-uqjlkp-MuiFormControl-root-MuiTextField-root .MuiInputBase-root input {
```
[the remark this answers is not in the document]
Comment (Jeyaprakash-NK) [reply style/base.css:206]: Discussed and deferred

File: style/batches.css [hunk 3]
```diff
@@ -103,7 +103,8 @@
   overflow-y: auto;
   max-height: 85vh;
 }
-.css-pjtntf-MuiInputBase-root-MuiOutlinedInput-root-MuiSelect-root .MuiInputBase-input { 
+.css-pjtntf-MuiInputBase-root-MuiOutlinedInput-root-MuiSelect-root
```
[the remark this answers is not in the document]
Comment (Jeyaprakash-NK) [reply style/batches.css]: Resolved

**Pull request merged**